.gitignore

```diff
@@ -2,5 +2,5 @@ target
 work
 *.iml
 .idea
-.bashrc
-setVersion.sh
\ No newline at end of file
+pom.xml.releaseBackup
+release.properties
```

Comment:
While at it, those final new lines 🐤
Your editor/IDE should be capable of ensuring it.

Reply:
Why would you not tell me eariler? :laughing:

Reply:
which part, fixing the final new lines or fixing your editor/IDE to do it for you? :trollface:

Reply:
I meant about fixing my IDE. :unamused:

Reply:
Sometimes it's good to suffer :sweat_smile: It might lead to the same result. Humans are lazy. The path of least resistance is always to think there must be a better way :laughing:

Reply:
Oh! Now I see it. 
![thank-you-bro-it-means-so-much](https://user-images.githubusercontent.com/23079344/58744018-964b8c00-8459-11e9-9f2b-bdd90d71e161.jpg)